pkg/operator/apiserver/controllerset/apiservercontrollerset.go

```diff
@@ -228,23 +228,20 @@ func (cs *APIServerControllerSet) WithWorkloadController(
 		cs.operatorClient,
 		kubeClient,
 		kubeInformersForNamespaces.PodLister(),
+		append(informers,
+			kubeInformersForNamespaces.InformersFor(targetNamespace).Core().V1().ConfigMaps().Informer(),
+			kubeInformersForNamespaces.InformersFor(targetNamespace).Core().V1().Secrets().Informer(),
+			kubeInformersForNamespaces.InformersFor(targetNamespace).Core().V1().Pods().Informer(),
+			kubeInformersForNamespaces.InformersFor(targetNamespace).Apps().V1().Deployments().Informer(),
+			kubeInformersForNamespaces.InformersFor(metav1.NamespaceSystem).Core().V1().Nodes().Informer(),
```

Comment:
this informer has a different namespace.

Reply:
so it wasn't used

Reply:
the first set of informers is expected to be non-namespaced so this would still get used, no?

Reply:
oh, yes, you are right :)